admin/client/src/legacy/GridField.js

```diff
@@ -133,21 +134,52 @@ $.entwine('ss', function($) {
 		}
 	});
 
+  $('.grid-field .action.action_import:button').entwine({
+    onclick: function(e) {
+      e.preventDefault();
+
+      var backdrop = $('.modal-backdrop');
+
+      if(backdrop.length < 1) {
+        backdrop = $('<div class="modal-backdrop fade in"></div>');
+        $('body').append(backdrop);
+      } else {
+        backdrop.addClass('fade in').fadeIn();
+      }
+
+      $(this.data('target')).addClass('in');
```

Comment:
save `$(this.data('target'))` to a variable, so you won't need to call it again below